app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
```

Comment:
created users registrations controller to customize the user sign up process of devise.